documentation/assemblies/metrics/assembly_metrics-cruise-control.adoc

```diff
@@ -0,0 +1,69 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='assembly-cruise-control-{context}']
+= Monitor Cruise Control
+
+If you are already using Prometheus and Grafana for monitoring of built-in Kafka metrics, you can configure Prometheus to also scrape the Cruise Control Prometheus endpoint.
+
+The example Grafana dashboard for the Cruise Control provides:
+
+* Information about rebalance proposals computation, goals violation, cluster balancedness and more
```

Comment:
This should be "optimization proposals" rather than "rebalance" to be consitent with the rest of the docs.